DESCRIPTION

```diff
@@ -1,39 +1,39 @@
-Package: shinyGovstyle
-Title: Custom Gov Style Inputs for Shiny
-Version: 0.1.0.9000
-Authors@R: c(
-    person("Ross", "Wyatt", , "ross.wyatt@justice.gov.uk", role = c("aut", "cre")),
-    person("Cameron", "Race", , "cameron.race@education.gov.uk", role = "ctb"),
-    person("Sarah", "Wong", role = "ctb"),
-    person("Richard", "Bielby", , "richard.bielby@education.gov.uk", role = "ctb"),
-    person("Charlotte", "Foster", , "charlotte.foster@education.gov.uk", role = "ctb"),
-    person("Jeni", "Martin", role = "ctb")
-  )
-Description: Collection of 'shiny' application styling that are the based
-    on the GOV.UK Design System.  See
-    <https://design-system.service.gov.uk/components/> for details.
-License: GPL (>= 3)
-URL: https://github.com/moj-analytical-services/shinyGovstyle
-BugReports: 
-    https://github.com/moj-analytical-services/shinyGovstyle/issues
-Depends: 
-    R (>= 3.5)
-Imports: 
-    htmltools,
-    jsonlite,
-    magrittr,
-    purrr,
-    reactable,
-    readr,
-    readODS,
-    shiny (>= 0.14),
-    shinyjs,
-    stringr,
-    writexl
-Suggests:
-    testthat,
-    shinytest2
-Encoding: UTF-8
-RoxygenNote: 7.3.2
-LazyData: true
-Config/testthat/edition: 3
+Package: shinyGovstyle
+Title: Custom Gov Style Inputs for Shiny
+Version: 0.1.0.9000
+Authors@R: c(
+    person("Ross", "Wyatt", , "ross.wyatt@justice.gov.uk", role = c("aut", "cre")),
+    person("Cameron", "Race", , "cameron.race@education.gov.uk", role = "ctb"),
+    person("Sarah", "Wong", role = "ctb"),
+    person("Richard", "Bielby", , "richard.bielby@education.gov.uk", role = "ctb"),
+    person("Charlotte", "Foster", , "charlotte.foster@education.gov.uk", role = "ctb"),
+    person("Jeni", "Martin", role = "ctb")
```

Comment:
ooo reminded me can we add Andy Baker in for his work on the reactables and value boxes please? Andrew.Baker1@dhsc.gov.uk :)

Reply:
Of course! Added 😄